macos/FluentUI/Button/Button.swift

```diff
@@ -185,6 +224,12 @@ open class Button: NSButton {
 			guard oldValue != isEnabled else {
 				return
 			}
+			if isEnabled {
```

Comment:
You can also use the ternary operator here:

`buttonState = isEnabled? .normal : .disabled`